auth0/src/main/java/com/auth0/android/authentication/AuthenticationAPIClient.java

```diff
@@ -318,6 +321,45 @@ public AuthenticationRequest loginWithOAuthAccessToken(@NonNull String token, @N
                 .addAuthenticationParameters(parameters);
     }
 
+    /**
+     * Log in a user using a token obtained from a Native Social Identity Provider, such as Facebook, using <a href="https://auth0.com/docs/api/authentication#token-exchange-for-native-social">'\oauth\token' endpoint</a>
+     * The default scope used is 'openid'.
+     * Example usage:
+     * <pre>
+     * {@code
+     * client.loginWithNativeSocialToken("{subject token}", "{subject token type}")
+     *      .start(new BaseCallback<Credentials>() {
+     *          {@literal}Override
+     *          public void onSuccess(Credentials payload) { }
+     *
+     *          {@literal}Override
+     *          public void onFailure(AuthenticationException error) { }
+     *      });
+     * }
+     * </pre>
+     *
+     * @param token     the subject token, typically obtained through the Identity Provider's SDK
+     * @param tokenType the subject token type that is associated with this Identity Provider. e.g. 'http://auth0.com/oauth/token-type/facebook-session-access-token'
+     * @return a request to configure and start that will yield {@link Credentials}
+     */
+    @SuppressWarnings("WeakerAccess")
+    public AuthenticationRequest loginWithNativeSocialToken(@NonNull String token, @NonNull String tokenType) {
```

Comment:
@lbalmaceda Where can I find the list of the supported `tokenTypes`. Is there one for Google SignIn?
From the doc (https://auth0.com/docs/api/authentication#token-exchange-for-native-social) it looks like only `apple-authz-code` is supported:
```
Identifier that indicates the type of subject_token. 
Currently supported native social values are: 
http://auth0.com/oauth/token-type/apple-authz-code.
```

Reply:
Google Sign-In is not supported yet, unfortunately. There's an ongoing effort to add Facebook Sign-In. The server-side team is working on this case by case per request from our customers. 
That link you posted will be updated when support for another provider is added. And of course, once it's available, we will update the SDKs.